crates/sui-json-rpc/src/read_api.rs

```diff
@@ -540,10 +540,12 @@ impl ReadApiServer for ReadApi {
             Some(self.state.get_executed_transaction(digest).await.tap_err(
                 |err| debug!(tx_digest=?digest, "Failed to get transaction: {:?}", err),
             )?);
+        println!("fetched transaction: {:?}", transaction);
```

Comment:
I am not sure if you are able to see the `println` result from log, why not just use `info!`?

Reply:
Let's also log the digest